File: Examples/Features/Formatting/Fonts/FontUnderline.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.Fonts
{
    public class FontUnderline : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            worksheet.Range["A1"].Value = "Single Underline";
            worksheet.Range["A1"].Font.Underline = UnderlineType.Single;
        }
    }
}

File: Examples.VB/Features/Formatting/Fonts/FontUnderline.vb
Namespace Features.Formatting.Fonts
    Public Class FontUnderline
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            worksheet.Range!A1.Value = "Single Underline"
            worksheet.Range!A1.Font.Underline = UnderlineType.Single
        End Sub
    End Class
End Namespace
